src/openms/source/DATASTRUCTURES/DataValue.cpp

```diff
@@ -236,10 +236,10 @@ namespace OpenMS
   }
 
   DataValue::DataValue(DataValue&& rhs) noexcept :
-    value_type_(std::move(rhs.value_type_)),
-    unit_type_(std::move(rhs.unit_type_)),
-    unit_(std::move(rhs.unit_)),
-    data_(std::move(rhs.data_))
+    value_type_(rhs.value_type_),
+    unit_type_(rhs.unit_type_),
+    unit_(rhs.unit_),
+    data_(rhs.data_)
```

Comment:
not sure if I am a big fan of this: the type of these variables could change and `std::move` does not hurt here but it shows intent more clearly

Reply:
agree